main/main/solution_n61_lightbulbs.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+LOCAL = os.environ.get("REMOTE_JUDGE", "false") != "true"
+
+if LOCAL:
+
+    class Node:
+        def __init__(self, value, left=None, right=None):
+            self.value = value
+            self.right = right
+            self.left = left
+
+
+def solution(root) -> int:
```

Comment:
write bfs or dfs algorithm as as separate class with 1 method. do not use recursion to skip stack length problems